tests/basic_map_operations_test.py

```diff
@@ -9,14 +9,13 @@ def setUp(self):
         pass
 
     def test_random_point(self):
-        for seed in [0, 1, 27, 29, 1939, 1982, 2015]:
-            random.seed(seed)
```

Comment:
This was very much unnecessary unless the RNG is expected to be of very dubious quality. I simplified it.